lib/device/Devices.hpp

```diff
@@ -0,0 +1,45 @@
+#ifndef __DEVICES_HPP__
+#define __DEVICES_HPP__
+
+#include <DeviceDescription.hpp>
+#include <Types.hpp>
+
+namespace Devices {
+
+const uint32_t RF_BOARD_MA_SUPPORTED = 2400 - 50;
+
+const DeviceDescription *SimpleRfBoardDescription(
+    uint8_t led_count, std::vector<StripFlag> flags) {
+  return new DeviceDescription(RF_BOARD_MA_SUPPORTED,
+                               {
+                                   new StripDescription(led_count, flags),
+                               });
+}
+
+const DeviceDescription *const bike = SimpleRfBoardDescription(30, {Bright});
```

Comment:
Can these be `constexpr`?

Reply:
I don't think so, at least the compiler complains a lot about it (even if I spread constexpr about)